### ISSUE TITLE
can't use screenshot directly in guest-agent after upgraded

### ISSUE THREAD

**Issue by renothing** [RocketChat/Rocket.Chat.Livechat#278]
with rocket.chat 1.3.0, guest-agent can paste their screenshot from clipboard (with Ctrl+V) in chat window.

but now it's unvaliable with Rocket.chat 1.3.2.

**Comment by renatobecker** [comment]
> with rocket.chat 1.3.0, guest-agent can paste their screenshot from clipboard (with Ctrl+V) in chat window.
> 
> but now it's unvaliable with Rocket.chat 1.3.2.

I've just tested it and everything looks fine.

**Comment by renothing** [comment]
my test enviroments:
rocket server: 1.3.2
browser: Version 77.0.3865.75 (Official Build) (64-bit)
livechat script :
```
<!-- Start of Rocket.Chat Livechat Script -->
<script type="text/javascript">
(function(w, d, s, u) {
	w.RocketChat = function(c) { w.RocketChat._.push(c) }; w.RocketChat._ = []; w.RocketChat.url = u;
	var h = d.getElementsByTagName(s)[0], j = d.createElement(s);
	j.async = true; j.src = 'https://example.com/packages/rocketchat_livechat/assets/rocketchat-livechat.min.js?_=201702160944';
	h.parentNode.insertBefore(j, h);
})(window, document, 'script', 'https://example.com/livechat');
</script>
<!-- End of Rocket.Chat Livechat Script -->
```

btw, the new version livechat script can't work anymore:

![TIM图片20190913152603](https://user-images.githubusercontent.com/261274/64844645-f21dd780-d63a-11e9-9c0a-4e216beaee8c.png)
![TIM图片20190913152535](https://user-images.githubusercontent.com/261274/64844651-f77b2200-d63a-11e9-8c99-e94472237634.png)
![TIM图片20190913152635](https://user-images.githubusercontent.com/261274/64844671-fe099980-d63a-11e9-89da-f567c92bc295.png)

**Comment by renothing** [comment]
fixed in 2.0
